Article/Article.md

```diff
@@ -17,89 +17,41 @@ abstract: |
 
 **Keywords:** replication study, gender differences, economic preferences
 
-
 # 1. Introduction
 
-Gender differences are nowadays extensively used as arguments and counter-arguments for decision and policy making, and the differences concerning the economic behaviors, such as happiness [@SPSU], competition [@CG; @GLL], or work preferences [@BG], are being studied in many sectors of the economy and economy-related fields.
+Gender differences concerning the economic behaviors, such as happiness [@SPSU], competition [@CG; @GLL], or work preferences [@BG], are being studied in sociology and economics. 
```

Comment:
Gender differences concerning economic behaviours, such as happiness [@SPSU], competition [@CG; @GLL], or work preferences [@BG], have been studied in sociology, psychology and economics for many decades now.

Reply:
Something missing here to start the discussion